### PR TITLE
Implemented Pet edit-update

### PR DESCRIPTION
**Pull request by albertoT75**
Please review, the code works.

### CODE REVIEW

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class PetsController < ApplicationController
+
+def edit
+  @user = current_user
+  @pet = Pet.find(params[:id])
+end
+
+def show
+  @pet = Pet.find(params[:id])
```
Comment (MarynaG) [inline app/controllers/pets_controller.rb]: Maybe it makes sense to create a method `set_pet` and call it before the methods `edit`, `show` and others if we need?

**Comment by albertoT75** [comment]
Please review

File: app/controllers/pets_controller.rb (hunk 1 above)
Comment (albertoT75) [reply app/controllers/pets_controller.rb]: Issues fixed

**Comment by Elena-Learning** [comment]
Looks good to me. But please have a look at the routes

**Comment by albertoT75** [comment]
Router fixed

**Comment by albertoT75** [comment]
Updated application policy

File: app/policies/pet_policy.rb [hunk 2]
```diff
@@ -4,4 +4,16 @@ def resolve
...
+    record.user == user
+  end
+
+  def show?
+    true
+  end
+
+  def edit?
+    update?
+  end
```
Comment (0math) [inline app/policies/pet_policy.rb]: It can be removed as edit inherits from ApplicationPolicy. And in there it checks for create

**Pull request merged**